tirgei/stack-overflow-lite
# #162574549 A user can view posted answers for a question

#### What does this PR do?
This Pull Request launches the page from which a user can view answers to a question

#### Description of Task to be completed?
The tasks for this PR is:
- Create the answers page
- Style the answers page

#### How should this be manually tested?
Clone this repository on your machine:
```$ git clone https://github.com/tirgei/stack-overflow-lite.git ```

CD into the project folder on your machine
```$ cd stack-overflow-lite ```

Check out the branch for this PR
``` $ git checkout ft-view-answers-162574549 ```

Open the `index.html` file on your browser then click on any question

### Pivotal Tracker story associated with this PR
[#162574549](https://www.pivotaltracker.com/story/show/162574549)

#### Screenshots
![ques](https://user-images.githubusercontent.com/17829576/50045623-2746ca80-00a7-11e9-9136-ea683e447b91.png)

![ques2](https://user-images.githubusercontent.com/17829576/50045624-2dd54200-00a7-11e9-8576-90ccb213a887.png)
